libp2p/wire.nim

```diff
@@ -152,7 +152,7 @@ proc createStreamServer*[T](ma: MultiAddress,
     raise newException(LPError, exc.msg)
 
 proc createAsyncSocket*(ma: MultiAddress): AsyncFD
-  {.raises: [LPError].} =
+  {.raises: [ValueError, LPError].} =
```

Comment:
why isn't it necessary before?

Reply:
@lchenut ?

Note that we don't really care about this proc, it's only used in the go-libp2p-daemon which is only used in CI

Reply:
An insight. I remove it.

Reply:
Well, it appears that it wasn't an insight, I don't know why it works locally but the CI fails:
/home/runner/work/nim-libp2p/nim-libp2p/libp2p/wire.nim(170, 35) Error: tryValue(protoCode(tryValue(ma[1]))) can raise an unlisted exception: ref ResultError[system.string]